src/ui/atoms/textField/TextField.tsx

```diff
@@ -20,48 +20,28 @@ export type TextFieldProps = InputBaseProps & {
    * If true, the TextField will take 100% of its parent's size.
    */
   readonly fullWidth?: boolean
```

Comment:
If you don't need to overload this property on the `TextField` component (and I think you don't), this must be removed as `fullWidth` is already spread with `InputBase` props.